dev/scripts/entrypoint_user.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Check if an environment variable exists and print its value
+if [ -n "$hf_token" ]; then
+  echo "The hf_token environment variable is: $hf_token"
```

Comment:
should be HF_TOKEN ?

Reply:
Removed the corresponding HF_TOKEN Follow the latest PR

Reply:
please remove any HF_TOKEN in CI code and build log. Otherwise, it will be warned by security team.